<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smu1_ida</class>
 <widget class="QDialog" name="smu1_ida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>SMU41_1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>IDa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>relative</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>absolute</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>70</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM1</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>120</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM2</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>170</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM3</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>220</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM4</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>270</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:StopM5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>70</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>120</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>170</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>220</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>270</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>details</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>10</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Stop All</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>310</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>more</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LsnM5</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:LspM5</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m5Axis" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM5</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM5</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m5Axis2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M5 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM5</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsPosM5</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m4Axis" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM4</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM4</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m4Axis2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M4 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM4</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM4</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m3Axis" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM3</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM3</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m3Axis2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M3 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM3</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM3</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m2Axis" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM2</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM2</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m2Axis2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M2 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM2</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM2</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m1Axis" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdRelM1</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:RelM1</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="m1Axis2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>M1 Left Rz</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>SMU41_1:rdPosM1</string>
   </property>
   <property name="units" stdset="0">
    <string>steps</string>
   </property>
   <property name="pv" stdset="0">
    <string>SMU41_1:AbsM1</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
